dev/PushNotifications/PushNotificationBackgroundTask.cpp

```diff
@@ -23,8 +29,17 @@ namespace winrt
 void PushNotificationBackgroundTask::Run(winrt::IBackgroundTaskInstance const& taskInstance)
 {
     auto appProperties = winrt::CoreApplication::Properties();
-    winrt::PushNotificationReceivedEventArgs activatedEventArgs = winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(taskInstance);
-    appProperties.Insert(ACTIVATED_EVENT_ARGS_KEY, activatedEventArgs);
+    if (PushNotificationHelpers::IsPackagedAppScenario())
```

Comment:
Add a comment here that "BackgroundTask::Run can either be activated by the OS background Infrastructure component or the Long Running Process"